website/source/docs/agent/http/agent.html.markdown

```diff
@@ -221,6 +235,19 @@ The return code is 200 on success.
 
 ### <a name="agent_force_leave"></a> /v1/agent/force-leave/\<node\>
 
+This endpoint is hit with a GET and is used to trigger a graceful leave and shutdown
```

Comment:
Let's change this and the other leave docs to say PUT even though it takes any verb.